.github/workflows/coverity.yml

```diff
@@ -1,6 +1,18 @@
 name: Coverity
 on:
   workflow_dispatch:
+    inputs:
+      description:
+        description: 'Custom build description'
+        required: true
```

Comment:
do we really need to have `required: true` if there is a default value?

Reply:
> do we really need to have `required: true` if there is a default value?

Default value is what is provided in web UI - you can drop it completely and without `required: true` an empty string will be allowed which I would prefer to avoid